ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -326,6 +452,25 @@ gcp_file_size(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
     return 0;
 }
 
+static int
+gcp_file_read(
+  WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t offset, size_t len, void *buf)
+{
+    WT_UNUSED(session);
+
+    gcp_file_handle *gcp_fh = reinterpret_cast<gcp_file_handle *>(file_handle);
+    gcp_file_system *fs = gcp_fh->file_system;
+
+    int ret;
+
+    if ((ret = fs->gcp_conn->read_object(gcp_fh->name, offset, len, buf)) != 0)
+        std::cerr << "gcp_file_read: read attempt failed." << std::endl;
+    else
+        std::cerr << "gcp_file_read: read succeeded." << std::endl;
```

Comment:
Do we want this output? We don't print "success" output elsewhere.

Reply:
removed here: [1487000](https://github.com/wiredtiger/wiredtiger/pull/8777/commits/1487000e645e0a2290646fdcf28759a1dab5ce96)